user_guides/configuringJenkins.md

```diff
@@ -0,0 +1,31 @@
+# How to download and install Jenkins
+
+## Why Jenkins 
+
+Based on our recent meeting with Andrew, we have decided that the best solution is to create a dedicated CI/CD tool because only using built in GCP tools is not user friendly for future students. It is best for all deployments to GCP be centralised. I believe Jenkins is probably the best solution as it is widely adopted, has GCP plugins and is OSS. I believe that success will follow these steps
```

Comment:
The best solution to what?

 I think you can also remove the part about Andrew. Think about this documentation as for someone coming in with no context. What do they need to know? What is GCP?

Reply:
Ok will do. I more made it specifically for our team so I thought the context was important. Will remove.